src/redux/actions/assetRegisterApi.js

```diff
@@ -183,35 +183,53 @@ export const getAsset = (url) => ({
 /**
  * Update an asset. Takes an asset resource object. The asset url must be non-null.
```

Comment:
could you add "Returns a function and requires the redux-thunk middleware." 
(and below)